src/refiners/training_utils/config.py

```diff
@@ -155,6 +155,7 @@ class ModelConfig(BaseModel):
     betas: tuple[float, float] | None = None
     eps: float | None = None
     weight_decay: float | None = None
+    train: bool = False
```

Comment:
I don't like this "train" attribute ; even when set to False, the model will still train since the learnable_parameter registration works anyway